### DISCUSSION TITLE
Service accounts cannot invite attendees without Domain-Wide Delegation of Authority.

### DISCUSSION THREAD

**Discussion by alvarocs92** [spatie/laravel-google-calendar#252] [Q&A]
hello, I have a problem with the library, it turns out that I carried out all the corresponding steps and when creating the example event it throws me an error.
does anyone know if there is a solution?

```
    public function createEvent(Request $request){
       
       
        $event = new Event;
        $event->name = 'calendario de reservas';
        $event->startDateTime = Carbon::now();
        $event->endDateTime = Carbon::now()->addHour();
        $event->addAttendee([
            'email' => 'pruebita@gmail.com',
            'name' => 'pruebita',
            'comment' => 'Test DE calendario',
        ]);
        
        $optParams = [
            'sendNotifications' => true
        ];
        
        $event->save('insertEvent', $optParams);
        
        dd($event);

    }
}
```

![imagen](https://user-images.githubusercontent.com/32555631/167536239-b5606cfb-b604-4ffb-8869-4fc385974398.png)

**Comment by apmendoza** [comment]
Hello, 

Visit this link: https://stackoverflow.com/questions/60760959/google-calendar-api-service-account-error.

I hope it's useful for you
